src/main/java/org/benetech/servicenet/service/mapper/OrganizationMapper.java

```diff
@@ -16,21 +16,19 @@
     unmappedTargetPolicy = IGNORE)
 public interface OrganizationMapper extends EntityMapper<OrganizationDTO, Organization> {
 
-    @Mapping(source = "location.id", target = "locationId")
-    @Mapping(source = "location.name", target = "locationName")
     @Mapping(source = "replacedBy.id", target = "replacedById")
     @Mapping(source = "sourceDocument.id", target = "sourceDocumentId")
     @Mapping(source = "sourceDocument.dateUploaded", target = "sourceDocumentDateUploaded")
     @Mapping(source = "account.id", target = "accountId")
     @Mapping(source = "account.name", target = "accountName")
     OrganizationDTO toDto(Organization organization);
 
-    @Mapping(source = "locationId", target = "location")
     @Mapping(source = "replacedById", target = "replacedBy")
     @Mapping(source = "sourceDocumentId", target = "sourceDocument")
     @Mapping(source = "accountId", target = "account")
     @Mapping(target = "funding", ignore = true)
     @Mapping(target = "programs", ignore = true)
+    @Mapping(target = "locations", ignore = true)
```

Comment:
Are sure it is unnecessary?

Reply:
I think we should ignore it because OrgaizationDTO doesn't store locations info. I think it's not relevant now.

Reply:
ok